book/src/tutorial/qt-gui/build-setup.md

```diff
@@ -0,0 +1,144 @@
+# Building a simple Qt application with Rust
+
+CXX-Qt provides helper libraries that make it easy for the Rust build system to link to Qt.
+
+The goal for now is to create a simple program that just launches an Qt/QML application.
+
+✅ Create a new Rust application
+```console
+cargo new qt-gui
+cd qt-gui
+```
+
+✅ Add a simple QML main file in `qml/main.qml`
+```qml
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Window 2.15
+
+ApplicationWindow {
+    height: 480
+    title: qsTr("Hello World")
+    visible: true
+    width: 640
+}
+```
+
+✅ Add dependencies to the `Cargo.toml` file
+```toml
+[dependencies]
+rustagram2="2"
```

Comment:
assume you are adding this now, so you don't need to tell them to modify the cargo.toml later ? As it's not required for the simple hello world ( could be worth a note )

Reply:
Yes, exactly, I don't want to jump back to previous files too much, as that can quickly get confusing.

Don't know if the note is worth it, as it may just be confusing/more to read.
The readers will already have been introduced to `rustagram` in the earlier CLI tutorial, so they probably know that this is only for the image conversion.
Or they don't notice and just copy-pasta these dependencies, which is fine for the flow of the tutorial as well.